src/components/AllServices/AllServicesLink.tsx

```diff
@@ -34,7 +46,36 @@ const AllServicesLink = ({ href, title, isExternal, category, group }: AllServic
   };
 
   const isFavorite = !!favoritePages.find(({ pathname, favorite }) => pathname === href && favorite);
-  return (
+  return enableAllServicesRedesign ? (
+    <Content component={ContentVariants.p} className={classNames('chr-c-favorite-trigger')}>
```

Comment:
Looking at the latest changes I don't think we need content component here. We can use just Flex with ChromeLink and other content.

Reply:
Getting rid of Content messes with font sizes and spacing of the links

Reply:
Oh I see. Mhh, maybe we should wrap the ChromeLink with one Content component and line 70 with another. When I checked the console it was throwing DOM error of nesting DIV in a P element which is not correct.

Reply:
I fixed the issue. Flexitem was acting as a \<div> and Content as a \<p> so I got rid of content and fixed the styling.